src/annotator/delegator.js

```diff
@@ -31,13 +31,18 @@ export default class Delegator {
    * @param {Object} [config]
    */
   constructor(element, config) {
-    // Some subclasses rely on a legacy mechanism of defining options at
-    // the class level by defining an `options` property on the prototype.
-    const classOptions = this.options || {};
-    this.options = { ...classOptions, ...config };
-    this.element = $(element);
+    this.options = { ...config };
+    this.element = element;
 
-    this.on = this.subscribe;
+    const el = /** @type {any} */ (element);
+    let eventBus = el._hypothesisEventBus;
+    if (!eventBus) {
+      eventBus = new EventEmitter();
+      el._hypothesisEventBus = eventBus;
+    }
+
+    this._eventBus = eventBus;
+    this._subscriptions = [];
```

Comment:
I wonder if this could use a typedef somewhere as its structure is depended on in `subscribe`, `unsubscribe`, and `destroy`...

Reply:
Yep, I'll add this.